hw/vendor/patches/pulp_riscv_dbg/0002-Use-lowrisc-instead-of-PULP-primitives.patch

```diff
@@ -77,15 +77,15 @@ index f131392..b7fc5e1 100644
 +    .clk_i   ( clk_i                ),
 +    .rst_ni  ( dmi_rst_ni           ), // reset only when system is re-set
 +    .clr_i   ( 1'b0                 ),
-+    .wdata_i ( resp_queue_data      ),
-+    .wvalid_i( dmi_req_valid_i      ),
-+    .wready_o( dmi_req_ready_o      ),
-+    .rdata_o ( dmi_resp_o.data      ),
-+    .rvalid_o( dmi_resp_valid_o     ),
-+    .rready_i( dmi_resp_ready_i     ),
-+    .depth_o (                      )  // Doesn't use
++    .wdata   ( resp_queue_data      ),
++    .wvalid  ( dmi_req_valid_i      ),
++    .wready  ( dmi_req_ready_o      ),
++    .rdata   ( dmi_resp_o.data      ),
++    .rvalid  ( dmi_resp_valid_o     ),
++    .rready  ( dmi_resp_ready_i     ),
++    .depth   (                      )  // Doesn't use
```

Comment:
Could it be that Scott's recent cleanup didn't update the repo clone, but only the patch?

Reply:
yeah that could be it. let me fix that directly in the patch repo.